block_printer:
  motion_commander:
    manipulator:
      command_type: LIN
      pose_reference: base_link
      pose_type: EulerFrame
      initial_pose: {position: [0.2,0,0.35], orientation: [0,0,0]}
      finish_pose: {position: [0.15,0,0.45], orientation: [0,0,0]}
      velocity: {type: "%", high: [100], low: [20]}
      acceleration: {type: "%", high: [100,100,100], low: [100,100,100]}
      force_threshold: {type: "%", high: [100,100,100], low: [100,100,70]}
    gripper:
      command_type: gripper
      poses: [0, 0.038, 0.0324]
      margin: -0.001
      effort_type: "N"
      effort: 50
